backend/app/tests/api/routes/test_prompts.py

```
@@ -0,0 +1,277 @@
+    assert prompts[1]["name"] == "prompt_1"
+    assert all(prompt["project_id"] == project_id for prompt in prompts)
+
+
+def test_get_prompts_route_empty(
+    client: TestClient,
+    db: Session,
+    user_api_key: APIKeyPublic,
+):
+    """Test retrieving an empty list when no prompts exist"""
+    response = client.get(
+        f"/api/v1/prompts/?skip=0&limit=100",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)